src/common/services/registerUserRequest.ts

```diff
@@ -0,0 +1,22 @@
+import { RegisterFormType } from '../../components/RegisterForm/RegisterForm.types';
+import { useMutation } from '@tanstack/react-query';
+
+const getRequestObject = (values: RegisterFormType) => {
+  return {
+    body: JSON.stringify(values),
+    headers: {
+      'Access-Control-Allow-Headers': 'Origin, X-Requested-With, Content-Type, Accept',
+      'Access-Control-Allow-Origin': '*',
+      'Content-Type': 'application/json',
+    },
+    method: 'POST',
+  };
+};
+
+const createUser = (values: RegisterFormType) => {
+  return fetch(`${import.meta.env.VITE_API_URL}/users`, getRequestObject(values));
+};
+
```

Comment:
missing return type now this will be `any` + this function is asynchronous so make it `async` please and await returned promise

Reply:
Changed in 8eac28f